mmv1/products/cloudrunv2/api.yaml

```diff
@@ -0,0 +1,717 @@
+                - !ruby/object:Api::Type::String
+                  name: "workingDir"
+                  description: |-
+                    Container's working directory. If not specified, the container runtime's default will be used, which might be configured in the container image.
+                - !ruby/object:Api::Type::NestedObject
+                  name: "livenessProbe"
+                  description: |-
+                    Periodic probe of container liveness. Container will be restarted if the probe fails. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                        name: "initialDelaySeconds"
+                        description: |-
+                          Number of seconds after the container has started before the probe is initiated. Defaults to 0 seconds. Minimum value is 0. Maximum value for liveness probe is 3600. Maximum value for startup probe is 240. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes
+                        default_value: 0
+                    - !ruby/object:Api::Type::Integer
+                        name: "timeoutSeconds"
+                        description: |-
+                          Number of seconds after which the probe times out. Defaults to 1 second. Minimum value is 1. Maximum value is 3600. Must be smaller than periodSeconds. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes
+                        default_value: 1
+                    - !ruby/object:Api::Type::Integer
+                        name: "periodSeconds"
+                        description: |-
+                          How often (in seconds) to perform the probe. Default to 10 seconds. Minimum value is 1. Maximum value for liveness probe is 3600. Maximum value for startup probe is 240. Must be greater or equal than timeoutSeconds
+                        default_value: 10
+                    - !ruby/object:Api::Type::Integer
+                        name: "failureThreshold"
+                        description: |-
+                          Minimum consecutive failures for the probe to be considered failed after having succeeded. Defaults to 3. Minimum value is 1.
+                        default_value: 3
+                    - !ruby/object:Api::Type::NestedObject
+                        name: "httpGet"
+                        description: |-
+                          HTTPGet specifies the http request to perform. Exactly one of HTTPGet or TCPSocket must be specified.
```

Comment:
I assume grpc is not supported yet?

Reply:
yeah, it doesn't look like it's supported in v2 based on the doc